### PR TITLE
Update with favicon

### PR DESCRIPTION
**Pull request by KirillMitskevich**

### CODE REVIEW

**Comment by TeatroIO** [comment]
I've prepared a stage to preview changes. <a href='http://add-favicon-with-config.datarockets-datarockets-com-224ddd7a6d7d5b5dacbe.ttrcloud.com/' target='_blank'>Open stage</a> or <a href='http://add-favicon-with-config.datarockets-datarockets-com-224ddd7a6d7d5b5dacbe.ttrcloud.com//teatro.log' target='_blank'>view logs</a>.

**Comment by Proghat** [comment]
@KirillMitskevich, what do you think about my [comment on Trello](https://trello.com/c/xR6iernU/176-when-i-visit-the-site-i-want-to-see-nice-favicon-on-browser-s-tab-so-i-can-easily-recognize-tab-on-tab-pane#comment-56afb0931ea10e1018878714):
> Let's try to create small favicon with just letter "d" on white background (square with probably rounded a little bit corners) or without background at all.
>
> For big squares let's use "datarockets" in the center.

Can you please do it? We used Helvetica for logo so it would be a good choice for favicon too in my opinion. At least for now.

File: app/assets/images/favicon/safari-pinned-tab.svg [hunk 1]
```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" standalone="no"?>
+<!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 20010904//EN"
+ "http://www.w3.org/TR/2001/REC-SVG-20010904/DTD/svg10.dtd">
+<svg version="1.0" xmlns="http://www.w3.org/2000/svg"
+ width="350.000000pt" height="350.000000pt" viewBox="0 0 350.000000 350.000000"
+ preserveAspectRatio="xMidYMid meet">
+<metadata>
+Created by potrace 1.11, written by Peter Selinger 2001-2013
```
Comment (ezhlobo) [inline app/assets/images/favicon/safari-pinned-tab.svg]: :rat

Comment (KirillMitskevich) [inline app/assets/images/favicon/safari-pinned-tab.svg]: gem генерит

**Pull request merged**